tests/monitoring_test.go

```diff
@@ -0,0 +1,135 @@
+package tests_test
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+const (
+	dataVolumeName = "test-dv"
+	validURL       = "http://www.example.com/example.img"
```

Comment:
i think it is a valid url, just as in `api_validation_test.go`

Reply:
ahh I see, that test module tests for invalid URL structure. I think in the monitoring module it may make sense to be called
nonExistentImageURL or something like that (the 404 is what triggers the restarts of importer)